assets/app/scripts/services/buildConfigs.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory('BuildConfigsService', function(DataService, $filter){
```

Comment:
I'm OK with a BuildsService that handles both build configs and builds. If we do create a BuildConfigsService, I'd want to look at what's already in BuildsService and move things as necessary.